# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/brscan/hotfix-def.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- brscan-1.0.1/libsane-brother/brother_modelinf.h.vanilla	2026-09-08 20:55:10.487116158 +0300
+++ brscan-1.0.1/libsane-brother/brother_modelinf.h	2026-09-08 20:56:50.102605453 +0300
@@ -29,6 +29,10 @@
 ;==============================================================================
 */
 
+#ifndef MUST_CONVERT_MODEL
+#define         MUST_CONVERT_MODEL      10
+#endif
+
 #ifndef _H_BROTHER_MODELINF
 #define _H_BROTHER_MODELINF
 

